src/docs/operator-guides/flex-operator-resources/index.md

```diff
@@ -63,12 +68,12 @@ https://www.sharetribe.com/docs/guides/how-to-deploy-ftw-to-production/
 
 ## Developer Resources
 
-Flex Documentation: https://www.sharetribe.com/docs/
-Flex API Reference: https://www.sharetribe.com/api-reference/
-FTW-daily Repo: https://github.com/sharetribe/flex-template-web
-FTW-hourly Repo: https://github.com/sharetribe/ftw-hourly
-Flex Developer Slack Channel: https://www.sharetribe.com/flex-slack
-Flex Support Email: flex-support@sharetribe.com
+Flex Documentation: https://www.sharetribe.com/docs/ Flex API Reference:
```

Comment:
Line changes in this file seem wrong after this change. These can be accessed at https://deploy-preview-363--sharetribe-flex-docs-site.netlify.app/docs/operator-guides/ and seem wrong. There are multiple lines that seem to be wrongly concatenated now in this file.

Reply:
Hmm, definitely not nicely formatted, but to me they look exactly the same as in the production deployment before this change. For example:

https://deploy-preview-363--sharetribe-flex-docs-site.netlify.app/docs/operator-guides/flex-operator-resources/
https://www.sharetribe.com/docs/operator-guides/flex-operator-resources/

Is there a difference?

Reply:
Yeah, you are correct. No difference, the formatting has been off earlier already. It looks ok in the code earlier, but it's off.

Reply:
Yep, markdown requires trailing whitespace to enforce a line break... ain't that great.

I'll clean up those and make them into lists etc.